xbmc/cores/RetroPlayer/rendering/RenderContext.cpp

```diff
@@ -70,16 +70,34 @@ void CRenderContext::ApplyStateBlock()
   m_rendering->ApplyStateBlock();
 }
 
-void CRenderContext::EnableGUIShader()
+#if defined(HAS_GL) || defined(HAS_GLES)
+namespace
+{
+static ESHADERMETHOD TranslateShaderMethod(GL_SHADER_METHOD method)
+{
+  switch (method)
+  {
+  case GL_SHADER_METHOD::DEFAULT: return SM_DEFAULT;
```

Comment:
you seem to hate those guys doing source code debugging.

Reply:
Snarky comments don't help. He means that when you put a `return` on the same line, you can't set a breakpoint. If this was changed to:

```c++
case GL_SHADER_METHOD::DEFAULT:
  return SM_DEFAULT;
```

then a breakpoint could be set on the `return` line.